app/components/review_table.rb

```diff
@@ -0,0 +1,11 @@
+class ReviewTable < ActionView::Component::Base
```

Comment:
I think the convention is to append `Component` to these, so `ReviewTableComponent`:

https://github.com/github/actionview-component#building-components